### PR TITLE
Add Docker support for local development

### PR DESCRIPTION
**Pull request by tagliala**

### CODE REVIEW

**Comment by vjt** [comment]
I understand this can be helpful, so feel free to update it and submit for review.

Thanks

File: docker/colore/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+WORKDIR $APP_DIR
+
+# Create application user and add permissions
+RUN groupadd --gid 1000 colore && \
+    useradd --uid 1000 --gid colore --shell /bin/bash --create-home colore && \
+    chown -R colore:colore $APP_DIR && \
+    chmod -R 755 $APP_DIR
+
+# Switch to application user
+USER colore
```
Comment (tagliala) [inline docker/colore/Dockerfile:48]: I read that it is not good to run Docker applications as root. Any improvement of this is appreciated

https://medium.com/@mccode/processes-in-containers-should-not-run-as-root-2feae3f0df3b

File: docker/colore/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    bundle config set deployment 'true' && bundle --retry 10; \
+  fi
+
+COPY --chown=colore:colore bin ./bin
+COPY --chown=colore:colore config ./config
+COPY --chown=colore:colore lib ./lib
+COPY --chown=colore:colore spec ./spec
+COPY --chown=colore:colore config.ru .
+COPY --chown=colore:colore Rakefile .
+COPY --chown=colore:colore unicorn.rb .
```
Comment (tagliala) [inline docker/colore/Dockerfile:76]: Not copying everything to limit the container size at the minimum

File: docker/colore/variables.env.example [hunk 3]
```diff
@@ -0,0 +1,4 @@
+LANG=en_US.UTF-8
+LANGUAGE=en_US.UTF-8
+LC_ALL=C.UTF-8
```
Comment (tagliala) [inline docker/colore/variables.env.example:3]: Maybe with #93 these settings are redundant

File: docker/colore/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,86 @@
+FROM ruby:2.6.5
```
Comment (tagliala) [inline docker/colore/Dockerfile:1]: This is using Debian

There is a centos 7 official image with Ruby 2.6, but it is not locked to 2.6.5

Ref: 
- https://hub.docker.com/layers/centos/ruby-26-centos7/latest/images/sha256-3827a0a4f348a3f9815d1f8d123eafbe129cdf57c7f8e0ed6265270d286cae80?context=explore
- https://hub.docker.com/_/ruby?tab=description&page=1&ordering=last_updated

**Comment by tagliala** [comment]
Merging after 4 years, this "works"

**Pull request merged**